habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -406,6 +422,21 @@ class ObjectSegmentationSensorConfig(LabSensorConfig):
     dimensionality: int = 256
 
 
+@attr.s(auto_attribs=True, slots=True)
+class StartRecepSegmentationSensorConfig(ObjectSegmentationSensorConfig):
```

Comment:
i don't really understand what we're doing here

Reply:
This is some weird hydra stuff that Habitat does. I also dont understand the reason for this extra step